operations/pyroscope/helm/pyroscope/ci/micro-services-values.yaml

```diff
@@ -27,10 +45,23 @@ pyroscope:
       replicaCount: 3
       resources:
         limits:
-          memory: 12Gi
+          memory: 4Gi
         requests:
           memory: 256Mi
           cpu: 100m
+    store-gateway:
+      kind: StatefulSet
+      replicaCount: 3
+      persistence:
+      # The store-gateway needs not need persistent storage, but we still run it as a StatefulSet
+      # This is to avoid having blocks of data being
+        enabled: false
+      resources:
+        limits:
+          memory: 16Gi
```

Comment:
Should we make it lower as in case of ingester?

Reply:
Yes that was an oversight thanks for catching 👍